packages/console/src/consts/tenants.ts

```diff
@@ -1,6 +1,19 @@
+import { defaultTenantId, ossConsolePath } from '@logto/schemas';
+
+import { isCloud } from './cloud';
+
 const isProduction = process.env.NODE_ENV === 'production';
 
 export const adminTenantEndpoint =
   process.env.ADMIN_TENANT_ENDPOINT ??
   (isProduction ? window.location.origin : 'http://localhost:3002');
-export const getUserTenantId = () => window.location.pathname.split('/')[1];
+
+export const getUserTenantId = () => {
+  if (isCloud) {
+    return window.location.pathname.split('/')[1] ?? '';
+  }
+
+  return defaultTenantId;
+};
```

Comment:
Maybe we need to support debuging in local dev machine.

Reply:
will add later

Reply:
will try to dev cloud locally